conbencher/python/conbencher/machine_info.py

```diff
@@ -0,0 +1,301 @@
+import os
```

Comment:
Is the goal to eventually remove this from `conbench`?

Reply:
Yeah, this shouldn't live in conbench itself; it always need to run on the benchmarking machine, not the server. It will live in both till everything gets migrated and we can simplify conbench.

Reply:
:nice: